hammr/commands/image/image.py

```diff
@@ -202,7 +202,7 @@ def arg_deploy(self):
         doParser = ArgumentParser(prog=self.cmd_name + " deploy", add_help=True,
                                   description="Deploy an instance of a published image on the targeted cloud.")
         mandatory = doParser.add_argument_group("mandatory arguments")
-        mandatory.add_argument('--id', dest='pid', required=True,
+        mandatory.add_argument('--publish-id', dest='pid', required=True,
```

Comment:
When we discussed this point, we also talked about a short option "-p". Could you add it here ?